documentation/asciidoc/microcontrollers/c_sdk/quick_start.adoc

```diff
@@ -0,0 +1,156 @@
+
+Install CMake (at least version 3.13), and GCC cross compiler
+
+----
+$ sudo apt install cmake gcc-arm-none-eabi libnewlib-arm-none-eabi libstdc++-arm-none-eabi-newlib
+----
+
+Set up your project to point to use the Raspberry Pi Pico SDK
+
+Either by cloning the SDK locally (most common) :
+
+----
+$ git clone https://github.com/raspberrypi/pico-sdk.git
+----
+
+Copy `pico_sdk_import.cmake` from the SDK into your project directory
+
+Set `PICO_SDK_PATH` to the SDK location in your environment, or pass it (`-DPICO_SDK_PATH=`) to `cmake` later.
+
+Setup a `CMakeLists.txt` like:
+
+----
+cmake_minimum_required(VERSION 3.13)
+
+# initialize the SDK based on PICO_SDK_PATH
+# note: this must happen before project()
+include(pico_sdk_import.cmake)
+
+project(my_project)
+
+# initialize the Raspberry Pi Pico SDK
+pico_sdk_init()
+
+# rest of your project
+----
+
+Or with the Raspberry Pi Pico SDK as a submodule :
+
+Clone the SDK as a submodule called `pico-sdk`
+
+Setup a `CMakeLists.txt` like:
+
+----
+cmake_minimum_required(VERSION 3.13)
+
+# initialize pico-sdk from submodule
+# note: this must happen before project()
+include(pico-sdk/pico_sdk_init.cmake)
+
+project(my_project)
+
+# initialize the Raspberry Pi Pico SDK
+pico_sdk_init()
+
+# rest of your project
+----
+
+Or with automatic download from GitHub :
+
+Copy `pico_sdk_import.cmake` from the SDK into your project directory
+
+Setup a `CMakeLists.txt` like:
+
+----
+cmake_minimum_required(VERSION 3.13)
+
+# initialize pico-sdk from GIT
+# (note this can come from environment, CMake cache etc)
+set(PICO_SDK_FETCH_FROM_GIT on)
+
+# pico_sdk_import.cmake is a single file copied from this SDK
+# note: this must happen before project()
+include(pico_sdk_import.cmake)
+
+project(my_project)
+
+# initialize the Raspberry Pi Pico SDK
+pico_sdk_init()
+
+# rest of your project
+----
+
+Or by cloning the SDK locally, but without copying `pico_sdk_import.cmake`:
+
+----
+$ git clone this Raspberry Pi Pico SDK repository
+----
+
+Setup a `CMakeLists.txt` like:
+
+----
+cmake_minimum_required(VERSION 3.13)
+
+# initialize the SDK directly
+include(/path/to/pico-sdk/pico_sdk_init.cmake)
+
+project(my_project)
+
+# initialize the Raspberry Pi Pico SDK
+pico_sdk_init()
+
+# rest of your project
+----
+
+Write your code (see https://github.com/raspberrypi/pico-examples[pico-examples] or the https://datasheets.raspberrypi.com/pico/raspberry-pi-pico-c-sdk.pdf[Raspberry Pi Pico C/{cpp} SDK] book for more information)
+
+About the simplest you can do is a single source file (e.g. `hello_world.c`)
+
+----
+#include <stdio.h>
+#include "pico/stdlib.h"
+
+int main() {
+    setup_default_uart();
+    printf("Hello, world!\n");
+    return 0;
+}
+And add the following to your CMakeLists.txt:
```

Comment:
You probably want to tweak this so that the C src-file and the CMakelist are in separate code-blocks?

Reply:
Direct copy and paste from Graham's README. 🤷‍♂️

Reply:
Fixed by https://github.com/raspberrypi/documentation/pull/2743/commits/a093334047ce5b3b4fcd79d0714cc565101956c4 (Graham's README was correct)